params/default.go

```diff
@@ -19,6 +18,14 @@ func DefaultNetwork() Network {
 	return defaultNetwork
 }
 
+func SetDefaultNetwork(net Network) error {
+	if err := net.Validate(); err != nil {
+		return err
+	}
+	defaultNetwork = net
+	return nil
+}
+
```

Comment:
We should make this a Node Option. Writing to global will create issues going further and brakes further celestia as a library approach.

Reply:
Can we make an issue for this? It will change via refactoring, would rather not introduce new options.

Reply:
This sounds like something that should be put into the refactoring tasks then, I do not feel like it belongs here.

Reply:
I am fine with postponing this as long as you keep this part of the refactoring efforts you take.

Reply:
It belongs here because it breaks the encapsulation of params pkg that it was designed with, but again I am fine with postponing this if you promise to fix this later on.

Reply:
Yes I mean I definitely agree, just think it fits much better with the refactorings than here. Whatever I do here will be superseeded by the refactorings anyways

Reply:
I also dislike this so will make sure to include it in refactorings -- can even rebase current refactoring PR and include as a commit.

Reply:
rebase current refactoring PR on main once this is merged, is what i meant